docs/resources/aws_internet_gateway.md

```diff
@@ -0,0 +1,87 @@
+---
+title: About the aws_internet_gateway Resource
+---
+
+# aws\_internet\_gateway
+
+Use the `aws_internet_gateway` InSpec audit resource to test the properties of a single AWS internet gateway.
+This resource is available in InSpec AWS resource pack version **[1.12.0](https://github.com/inspec/inspec-aws/releases/tag/v1.12.0)** onwards.
+
+## Syntax
+
+An `aws_internet_gateway` resource block declares the tests for a single AWS internet gateway by id or name.
+
+    describe aws_internet_gateway(id: 'igw-abc0123456789deff') do
+      it { should exist }
+    end 
+
+    describe aws_internet_gateway(name: 'my-igw') do
+      it { should exist }
+    end
+
+#### Parameters
+
+Either the id or the name must be provided.
+
+##### id _(required if `name` not provided)_
+
+The value of the `internet_gateway_id` assigned by the AWS after the resource has been created.
+This should be in the format of `igw-` followed by 8 or 17 hexadecimal characters and passed as an `id: 'value'` key-value entry in a hash.
+
+##### name _(required if `id` not provided)_
+
+If a `Name` tag is applied to the internet gateway, this can be used to lookup the resource.
+This must be passed as a `name: 'value'` key-value entry in a hash.
+If there are multiple internet gateways with the same name, this resource will raise an error.
+
+## Properties
+
+|Property            | Description |
+| ---                | --- |
+|id                  | The ID of the internet gateway. |
+|name                | The value of the `Name` tag. It is `nil` if not defined. |
+|vpc\_id             | The ID of the attached VPC. It is `nil` if the resource is in a `detached` state. |
+|tags                | An hash with each key-value pair corresponding to a tag associated with the entity. |
```

Comment:
Typo: `A hash with`

Reply:
Perhaps this could be re-worded slightly. Maybe something along the lines of...
`A hash, with each key-value pair corresponding to an Internet Gateway tag`